src/App.js

```diff
@@ -1,12 +1,107 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import './App.css';
+import pokemon from "./assets/pokemon.png"
+import axios from "axios";
+import PokemonDetails from "./components/PokemonDetails";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [data, setData] = React.useState({});
```

Comment:
Als je hier de default waarde óók `undefined` maakt, dan kun je een wat simpelere check doen of het ingeladen is of niet